### DISCUSSION TITLE
vercel how to stop edge-functions stream?

### DISCUSSION THREAD

**Discussion by tianez** [vercel/community#2410] [Help] [answered]
### Summary

I create a streaming with edge-functions, but on the client side I can't stop the stream with the cancel method, The server is always running.

**Comment by smaeda-ks** [comment]
@tianez could you include more details? https://vercel.com/guides/creating-a-minimal-reproducible-example

**Reply by tianez** [reply]
[https://github.com/tianez/vercel-stream.git](url) the example
The client has been stopped,The server is always running.
![image](https://github.com/vercel/community/assets/6291579/ace7a8e6-dabe-43a4-93c0-46fe94e940d4)
![image](https://github.com/vercel/community/assets/6291579/bdd87065-e30b-4d9e-bc05-842c023f0ce5)

**Reply by smaeda-ks** [reply, marked as answer]
@tianez That's expected because you're testing it locally, and the behavior in how the instance is evicted differs from when you deploy it on Vercel. If you test it on your preview/production deployments, I believe it gets stopped, unless you do something with `waituntil`:
https://vercel.com/docs/concepts/functions/edge-functions/edge-functions-api#waituntil

**Reply by tianez** [reply]
deploy it on Vercel is ok, but build it and start it in my computer is still wrong.
